features/zones/Delete.feature

```diff
@@ -1,15 +1,15 @@
 Feature: zones / delete
+
   Background:
     Given the CSS selectors
-      | Alias          | Selector                                                       |
-      | items          | [data-testid="zone-cp-collection"]                             |
-      | item           | $items tbody tr                                                |
-      | actions-button | $item:nth-child(1) [data-testid='dropdown-trigger'] button     |
-      | delete-button  | $item:nth-child(1) [data-testid='dropdown-delete-item'] button |
-      | delete-prompt  | [data-testid="delete-zone-modal"]                              |
-      | confirm-button | $delete-prompt [data-testid='modal-action-button']             |
-      | confirm-input  | $delete-prompt [data-testid='confirmation-input']              |
-
+      | Alias          | Selector                                                   |
+      | items          | [data-testid="zone-cp-collection"]                         |
+      | item           | $items tbody tr                                            |
+      | actions-button | $item:nth-child(1) [data-testid='dropdown-trigger'] button |
+      | delete-button  | $item:nth-child(1) [data-testid='dropdown-delete-item']    |
```

Comment:
The change here is the removal of `button`, which is no longer required because the kongponent update adds any additional props to the button itself now

The rest of the change here is just formatting